Sdk/MoyasarSdk/Services/PaymentServices/PaymentServices.swift

```diff
@@ -72,6 +72,10 @@ final public class PaymentService {
             
             return try decoder.decode(T.self, from: data)
         } catch {
+            if let moyasarError = error as? MoyasarError {
```

Comment:
we may use this enhancement . however i see no need for it becasue both cases we throw moyasar error

catch let error as MoyasarError {
        throw error
    } catch {
        throw MoyasarError.unexpectedError("Request failed: \(error.localizedDescription)")
    }

Reply:
That looks much better 👍